ddocs/medic/_attachments/translations/messages-en.properties

```diff
@@ -1385,6 +1387,32 @@ user.place.help = All users must be associated with a place in the system to be
 user.username.help = This is what you will use to log in to the app.
 username.invalid = Invalid user name. Valid characters are lower case letters, numbers, underscore (_), and hyphen (-).
 username.taken = Username "{{username}}" already taken.
+users.manage.back = Back to user list
```

Comment:
- Please sort these keys alphabetically, we try to keep them that way, for example `users.manage.back` should go after `users.manage.add-single-user`
- Also please use underscore instead of dash to separate words. For example look at `instance.upgrade.no_new_releases`

This apply for the other lines below
Thanks!